### PR TITLE
Makes satchel wearable on belt slot

### PR DESCRIPTION
**Pull request by SynthTwo**
## About The Pull Request
Got this from Monke.

## Why it's Good for the Game

More storage options huzzah.

## Proof of Testing

https://github.com/user-attachments/assets/438856f7-fbac-4ad7-a628-0510ce651290

## Changelog

:cl:
code: Satchel can be worn on belt slot now.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -28,6 +28,16 @@
 	. = ..()
 	AddElement(/datum/element/attack_equip)
 
+/obj/item/storage/backpack/equipped(mob/user, slot, initial)
+	if(slot == ITEM_SLOT_BACK)
+		if(HAS_TRAIT(user, TRAIT_BELT_SATCHEL))
+			slowdown++
```
Comment (DGamerL) [inline code/game/objects/items/storage/backpack.dm:34]:
```suggestion
	if(slot == ITEM_SLOT_BACK && HAS_TRAIT(user, TRAIT_BELT_SATCHEL))
		slowdown++
```

File: code/game/objects/items/storage/backpack.dm [hunk 2]
```diff
@@ -295,6 +305,27 @@
...
+	. = ..()
+	if(slot == ITEM_SLOT_BELT)
+		ADD_TRAIT(user, TRAIT_BELT_SATCHEL, CLOTHING_TRAIT)
+		if(istype(user.get_item_by_slot(ITEM_SLOT_BACK), /obj/item/storage/backpack))
+			var/obj/item/storage/backpack/selected_bag = user.get_item_by_slot(ITEM_SLOT_BACK)
+			selected_bag.slowdown++
+
+/obj/item/storage/backpack/satchel/dropped(mob/user, silent)
+	. = ..()
+	if(HAS_TRAIT(user, TRAIT_BELT_SATCHEL))
```
Comment (DGamerL) [inline code/game/objects/items/storage/backpack.dm:324]: Both of these would be cleaner as a guard clause

File: code/game/objects/items/storage/backpack.dm [hunk 3]
```diff
@@ -295,6 +305,27 @@
...
+		REMOVE_TRAIT(user, TRAIT_BELT_SATCHEL, CLOTHING_TRAIT)
+		if(istype(user.get_item_by_slot(ITEM_SLOT_BACK), /obj/item/storage/backpack))
+			var/obj/item/storage/backpack/selected_bag = user.get_item_by_slot(ITEM_SLOT_BACK)
+			selected_bag.slowdown = initial(selected_bag.slowdown)
```
Comment (DGamerL) [inline code/game/objects/items/storage/backpack.dm:328]:
```suggestion
		var/obj/item/storage/backpack/selected_bag = user.get_item_by_slot(ITEM_SLOT_BACK)
		if(istype(selected_bag))		
			selected_bag.slowdown = initial(selected_bag.slowdown)
```

File: code/game/objects/items/storage/backpack.dm [hunk 4]
```diff
@@ -295,7 +306,27 @@
...
+			worn_backpack.slowdown = 1.5
+			user.update_equipment_speed_mods()
+
+/obj/item/storage/backpack/satchel/dropped(mob/user, silent)
+	. = ..()
+	REMOVE_TRAIT(user, TRAIT_BELT_SATCHEL, CLOTHING_TRAIT)
+	var/obj/item/storage/backpack/worn_backpack = user.get_item_by_slot(ITEM_SLOT_BACK)
+	if(istype(worn_backpack))
+		worn_backpack.slowdown = initial(worn_backpack.slowdown)
 /obj/item/storage/backpack/satchel/leather
```
Comment (DGamerL) [inline code/game/objects/items/storage/backpack.dm:330]:
```suggestion
		worn_backpack.slowdown = initial(worn_backpack.slowdown)
/obj/item/storage/backpack/satchel/leather

```
Do you not need to call `update_equipment_speed_mods()` here?

File: code/game/objects/items/storage/backpack.dm [hunk 5]
```diff
@@ -28,6 +28,17 @@
...
+	if(slot == ITEM_SLOT_BACK)
+		var/obj/item/storage/backpack/satchel/worn_satchel = user.get_item_by_slot(ITEM_SLOT_BELT)
+		if(istype(worn_satchel) && HAS_TRAIT(user, TRAIT_BELT_SATCHEL))
+			slowdown = 1.5
+			user.update_equipment_speed_mods()
+
+/obj/item/storage/backpack/dropped(mob/user, silent)
+	. = ..()
+	slowdown = initial(slowdown)
 /*
```
Comment (DGamerL) [inline code/game/objects/items/storage/backpack.dm:42]:
```suggestion
	slowdown = initial(slowdown)
/*

```

**Comment by KnigTheThrasher** [comment]
dont forget to mark the non-modular stuff

**Pull request merged**